packages/react-composites/src/composites/CallWithChatComposite/adapter/AzureCommunicationCallWithChatAdapter.ts

```diff
@@ -179,6 +181,24 @@ export class AzureCommunicationCallWithChatAdapter implements CallWithChatAdapte
   private originCallChatAdapter: ChatAdapter | undefined;
   private breakoutRoomChatAdapter: ChatAdapter | undefined;
 
+  private chatEventListeners: Map<string, Listener[]> = new Map();
```

Comment:
You can use a `Set` instead of array to ensure no duplicates and syntactically inform the reader that it is designed for uniqueness

Reply:
Your add/remove functions won't then need to recreate the array, but can just use `set.add` and `set.remove`